rules/.eslintrc

```diff
@@ -1,5 +0,0 @@
-{
```

Comment:
This was just a lint config for the folder, it's not part of our exported lint config. (Now do you see why I deleted them?)